aiounifi/interfaces/connectivity.py

```diff
@@ -71,6 +71,12 @@ async def login(self) -> None:
             "rememberMe": True,
         }
 
+        if hasattr(self.config, "site") and self.config.site is not None:
+            auth["site_name"] = self.config.site
+
+        if hasattr(self.config, "for_hotspot") and self.config.for_hotspot is not None:
+            auth["for_hotspot"] = self.config.for_hotspot
```

Comment:
```suggestion
        if self.config.for_hotspot:
            auth["for_hotspot"] = True
```
Based on other review comment we for_hotspot can only be a bool and then the only valid state to send the for_hostpot is when it is true if I understand its use case

Reply:
For the authentication as a hotspot manager to succeed, it is necessary to include both "site_name=[default]" and "for_hotspot=true".